LayoutTests/platform/gtk/fast/forms/form-in-malformed-markup-expected.txt

```diff
@@ -4,7 +4,7 @@ layer at (0,0) size 800x600
   RenderBlock {HTML} at (0,0) size 800x600
     RenderBody {BODY} at (8,8) size 784x584
       RenderBlock (anonymous) at (0,0) size 784x0
-        RenderInline {B} at (0,0) size 0x0
+        RenderInline {B} at (0,-14) size 0x17
```

Comment:
I'm not sure if this is right. Maybe @nt1m could confirm?

Reply:
I can make test to use old behavior by adding `* { unicode-bidi: normal }` like other tests. I didn't done it because I didn't notice rendering difference which is worse off like other cases.